File: Loops/Fizzbuzz.c
// Sasha Rinitra, FizzBuzz C
#include <stdio.h>
int num;
int main(void) {
  for(num=0; num<51; num++){
    if(num % 3){
        printf("Fizz\n");
    }else if(num % 5){
        printf("Buzz\n");
    }else if(num % 3 && num % 5){
        printf("FizzBuzz\n");
    }else{
        printf("%d\n", num);
    }
        
  }
  return 0;
}

File: Loops/FizzBuzz.py
#Sasha Rinitra, FizzBuzz Python
num = 0
for num in range(1,51):
    if num % 3 and num % 5:
        print("FizzBuzz")
    elif num % 3:
        print("Fizz")
    elif num % 5:
        print("Buzz")
    else:
        print(num)